style.css

```diff
@@ -455,8 +461,344 @@ ul.card-item {
   line-height: 32px;
   text-align: center;
   color: #42526e;
+  font-family: 'inter', sans-serif !important;
 }
 
-.work-section {
-  display: grid;
+@media screen and (max-width: 720px) {
```

Comment:
**You made a great effort on this project! I love it!**
_However, some places need to be fixed for the next step. If you are with me let's get started! 😄_

- [ ] You used media queries to make the website responsive. It's perfect! 
But, when it comes to width range it should be different. 
The value for the min-width property should be in the range of 768px or wider. Please follow this requirement!

Reply:
done

Reply:
done